devices/stm32u585.yaml

```
@@ -9,6 +9,58 @@ _modify:
   ADC12:
     name: ADC12_Common
 
+PWR:
+  _add:
+    CR4:
+      displayName: CR4
```

```suggestion
      displayName: PWR control register 4
```
Will apply all suggestions at the end.